### DISCUSSION TITLE
is the drop() function in migration schema will executed when run command adonis migration:rollback ?

### DISCUSSION THREAD

**Discussion by fmgono** [adonisjs/core#1623] [General]
is the drop() function in migration schema will executed when run command adonis migration:rollback ?
because whenever i run migration rollback, the table still exists,,,
i don't know why is it because i have an relation column ? so when execute drop function (which is drop table) , it will cause error because we executed drop table without 'cascade' ?

**Comment by thetutlage** [comment]
Mind sharing the code for the migration file?
